indexer/src/main/java/org/pdxfinder/utils/CreateLocalFeeds.java

```diff
@@ -103,15 +118,15 @@ public void run(String... args) throws Exception {
 
             createJAXFeeds();
 
-            createIRCCFeeds();
+          //  createIRCCFeeds();
```

Comment:
This commented out code should be removed

Reply:
https://github.com/PDXFinder/pdxfinder/commit/7002eb76912fbef17346a18365842855d1d5efae